Networking/Networking/Model/ShippingLabel/Packages/Carriers and Rates/ShippingLabelPackageSelected.swift

```diff
@@ -5,8 +5,8 @@ import Codegen
 ///
 public struct ShippingLabelPackageSelected: Equatable, GeneratedFakeable {
 
-    /// The id will be always "default_box"
-    public let id: String = "default_box"
+    /// The id will be always the name of the package.
```

Comment:
Nit: I don't think the ID is the same as the package's name?

Reply:
Fixed in the other PR on which I'm working on 👍